x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -281,36 +281,4 @@ export function trainedModelsRoutes({ router, routeGuard }: RouteInitialization)
       }
     })
   );
-
-  /**
-   * @apiGroup TrainedModels
-   *
-   * @api {get} /api/ml/trained_models/:modelId/deployment/_stats Get trained model deployment stats
-   * @apiName GetTrainedModelDeploymentStats
```

Comment:
If this route is going altogether, `GetTrainedModelDeploymentStats` should be removed from apidoc.json

Reply:
Done in efd220e2c554ffdb6a559f7c2e3154d3952e6eca